CryptoAnalysis/src/main/java/crypto/constraints/PredicateConstraint.java

```diff
@@ -155,6 +156,14 @@ private void evaluateHardCodedPredicate(CrySLPredicate hardCodedPredicate) {
 
 			Collection<ExtractedValue> extractedValues = context.getParsAndVals().get(cs);
 			for (ExtractedValue extractedValue : extractedValues) {
+				// Check if value for predicate could be extracted
+				/*if (extractedValue.getValue().equals(Val.zero())) {
+					Statement statement = cs.stmt();
+					ImpreciseValueExtractionError error = new ImpreciseValueExtractionError(hardCodedPredicate, statement, context.getSpecification());
+					errors.add(error);
+					continue;
+				} */
+
```

Comment:
Why is this added but commented out?

Reply:
Removed, this was part of debugging an error